wiki/content/howto/index.md

```diff
@@ -104,6 +106,20 @@ Debug the p directory, inspecting the history of a particular key:
 $ dgraph debug --postings ./p --lookup 00000b6465736372697074696f6e020866617374 --history
 ```
 
+Debug an encrypted p directory with the key in a local file at the path  ./key_file:
+
+```sh
+$ dgraph debug --postings ./p --keyfile ./key_file
+```
+
+
+{{% notice "note" %}}
+The key file contains the key used to decrypt/encrypt the db. This key should be kept secret. As a best practice, 
+
+- Do not store permanently the key in the local disk.  Delete it after using it for debug tool.
```

Comment:
Do not  store the key file on the disk  permanently. Back it up in a safe place and delete it after using it with the debug tool.